service/src/main/java/org/apache/celeborn/server/common/service/config/FsConfigServiceImpl.java

```diff
@@ -56,16 +57,22 @@ public synchronized void refreshAllCache() {
       List<Map<String, Object>> dynamicConfigs = yaml.load(fileInputStream);
       for (Map<String, Object> settings : dynamicConfigs) {
         String tenantId = (String) settings.get(CONF_TENANT_ID);
+        String userId = (String) settings.get(CONF_TENANT_USER_ID);
         String level = (String) settings.get(CONF_LEVEL);
         Map<String, String> config =
             ((Map<String, Object>) settings.get(CONF_CONFIG))
                 .entrySet().stream()
                     .collect(Collectors.toMap(Map.Entry::getKey, a -> a.getValue().toString()));
-        if (ConfigLevel.TENANT.name().equals(level)) {
-          TenantConfig tenantConfig = new TenantConfig(this, tenantId, null, config);
-          tenantConfs.put(tenantId, tenantConfig);
+        if (ConfigLevel.TENANT_USER.name().equals(level)) {
+          TenantConfig tenantConfig = new TenantConfig(this, tenantId, userId, config);
+          tenantConfs.put(tenantId + "." + userId, tenantConfig);
```

Comment:
May be we can save user config into a separate conf <UserIndetifier, TenantConf>, then if we want to filter by user name or list all tenantUser configs, that's would be more easy.